tests/bgp/test_bgpmon_v6.py

```diff
@@ -149,7 +196,7 @@ def bgpmon_peer_connected(duthost, bgpmon_peer):
                    local_asn=asn,
                    peer_asn=asn,
                    port=BGP_MONITOR_PORT, passive=True)
-    ptfhost.shell("ip neigh add %s lladdr %s dev %s" % (local_addr, asichost.get_router_mac(), ptf_interface))
+    ptfhost.shell("ip neigh add %s lladdr %s dev %s" % (local_addr, get_uplink_route_mac(duthosts), ptf_interface))
```

Comment:
Why do we need this change. 
get_uplink_route_mac will return the router mac for the first asic on the uplink LC. 
of this test is running on any other LC then router_mac should be asichost.get_router_mac() for one if the asic present in that LC?